### PR TITLE
More Jsdoc module fixes

### PR DESCRIPTION
**Pull request by ger-benjamin**
For https://github.com/openlayers/openlayers/issues/7970#issuecomment-386285758

I also update examples but I leave occurrences like: https://github.com/openlayers/openlayers/blob/master/src/ol/render/canvas/index.jsdoc#L2 (I guess ?)

### CODE REVIEW

File: src/ol/render/webgl/Replay.js [hunk 1]
```diff
@@ -114,7 +114,7 @@ const WebGLReplay = function(tolerance, maxExtent) {
   /**
    * Optional parameter for PolygonReplay instances.
    * @protected
-   * @type {ol.render.webgl.LineStringReplay|undefined}
+   * @type {module:/ol/render/webgl/LineStringReplay|undefined}
```
Comment (ahocevar) [inline src/ol/render/webgl/Replay.js]: Should be `module:ol/render/webgl/LineStringReplay` (without leading slash).

File: src/ol/format/readme.md [hunk 2]
```diff
@@ -112,23 +112,23 @@ Now consider the case of parsing multiple features:
...
 
 ### Common operations
 
 In the above there are many common operations, like setting the property of the object on the top of the stack, or reading an object and appending that to an array on the top of the stack. There are many helper functions here, for example:
 
-* `ol.xml.makeObjectPropertySetter` reads a value from the child element and sets a property on the object on the top of the stack.
-* `ol.xml.makeArrayPusher` reads a value from the child element and appends it to the array on the top of the stack.
-* `ol.xml.makeReplacer` reads a value from the child element and *replaces* whatever is on top of the stack with it.
+* `module:ol/xml~makeObjectPropertySetter` reads a value from the child element and sets a property on the object on the top of the stack.
+* `module:/ol/xml~makeArrayPusher` reads a value from the child element and appends it to the array on the top of the stack.
```
Comment (ahocevar) [inline src/ol/format/readme.md]: Should be `module:ol/xml~makeArrayPusher` (no leading slash).

**Comment by ger-benjamin** [comment]
Done.
Finally, I've not touched files in `doc/` in this PR. I'll do another to do that.

The only `ol.` notations I don't know what to do with, are these occurrences:
`src/ol/format/MVT.js: * @param {ol.ext.PBF} pbf The PBF.`

Should I do something with them ?

**Comment by ger-benjamin** [comment]
I've tried to rebase after #8166 and I can comfirm that all occurrences are fixed.
My PR is useless, I close.

**Pull request closed without merging**